### ISSUE TITLE
Jetpack Badge not read by TalkBack

### ISSUE THREAD

**Issue by thehenrybyrd** [wordpress-mobile/WordPress-Android#17092]
### Expected behavior
Jetpack badge should be read in TalkBack.

### Actual behavior
Jetpack badge is not read, though Jetpack banner is. Verified in "Home" tab, "Me", and Activity log. Suspect this is the case for Jetpack badge everywhere.

I discovered this in Activity log, where the Jetpack badge is not read when viewing individual events.

### Steps to reproduce the behavior
1. Open the WordPress app to a screen with the Jetpack badge (not banner).
2. Turn on TalkBack (press and hold the + and - volume buttons for 3 seconds).
3. Using TalkBack, navigate through the items on the screen.
4. Note that when you get to the Jetpack badge, it is skipped - neither highlighted nor read.

https://user-images.githubusercontent.com/15107387/186649579-7c0dbad8-5e55-4807-9ffd-e0dad12afb6c.mp4

##### Tested on Samsung Galaxy S21, Android 12, WPAndroid 20.5-rc-2

**Comment by johnoricha** [comment]
@thehenrybyrd I'm working on this issue

**Comment by ovitrif** [comment]
> @thehenrybyrd I'm working on this issue

Hey @johnoricha, thank you for taking the initiative to work on this issue 🙇 .

I was just planning to focus on this one a few hours ago; but I'm more than happy to encourage you to continue working on this and find another issue for me to fix.

Feel free to ask me anything if needed when you're starting on a fix!

I was involved with the development of the badge not long ago and at that time we've decided to disable TalkBack, then after a broader discussion the decision was to enable it again, and at that time the issue was registered here.

Best of luck on this and don't hesitate to ask me if you need anything with setting up the repo / looking into the issue.

I'd also be glad to review your pull request when it's available 👍

**Comment by johnoricha** [comment]
> > @thehenrybyrd I'm working on this issue
> 
> Hey @johnoricha, thank you for taking the initiative to work on this issue 🙇 .
> 
> I was just planning to focus on this one a few hours ago; but I'm more than happy to encourage you to continue working on this and find another issue for me to fix.
> 
> Feel free to ask me anything if needed when you're starting on a fix!
> 
> I was involved with the development of the badge not long ago and at that time we've decided to disable TalkBack, then after a broader discussion the decision was to enable it again, and at that time the issue was registered here.
> 
> Best of luck on this and don't hesitate to ask me if you need anything with setting up the repo / looking into the issue.
> 
> I'd also be glad to review your pull request when it's available 👍 
> 
> 

@ovitrif thank you. I'll reach out if I encounter any issues
